packages/dialtone-css/lib/build/less/components/avatar.less

```
@@ -372,6 +373,8 @@
       }
 
       > .d-avatar__count {
+        padding-right: var(--dt-space-0);
+        padding-left: var(--dt-space-0);
```

Even though `next` has the big refactor to update all to Logical Properties, new or updates should use logical. In this case, shorthand is...
```suggestion
        padding-inline: var(--dt-space-0);
```